extensions/intellij/intellij_generator_plugin/src/main/java/com/bloc/intellij_generator_plugin/action/BlocStatePackage.java

```diff
@@ -0,0 +1,11 @@
+package com.bloc.intellij_generator_plugin.action;
+
+/**
+ * @author Martin Neff, approppo GmbH (martin.neff@approppo.ch)
+ * @since 13.09.22
+ */
```

Comment:
```suggestion
```

Is this necessary? I'd prefer not to include such comments in the codebase because they don't add much value and anyone can look at git blame for a history of changes and the contributor.

Reply:
Ah i forgot to remove the file header in Intellij. I'll remove all those comments